### ISSUE TITLE
Attempting to access a deleted package through a deep link results in a default error screen

### ISSUE THREAD

**Issue by Lycanic** [chirun-ncl/chirun-lti#30]
When a package linked on Canvas is deleted, attempting to access it results in the returning of the DoesNotExist error to the user and no ability to re-link to a new package.

We should probably tell the user that the package has been deleted (and/or verify whether there are other sources of this issue) and allow them to choose a new package.

**Comment by christianp** [comment]
Yeah, I think that deleted packages should disappear completely, so the response should be the same as if they never existed.
